<launch>

   <include file="$(find pr2_arm_navigation_config)/launch/pr2_planning_environment.launch" />

   <node machine="c2" pkg="pr2_arm_kinematics_constraint_aware" type="pr2_arm_kinematics_constraint_aware" name="pr2_right_arm_kinematics" output="screen">
   <!-- IK parameters -->
   <param name="tip_name" value="r_wrist_roll_link" />
   <param name="root_name" value="torso_lift_link" />

   <!-- Collision environment parameters -->
   <remap from="robot_description" to="/robot_description" />
   <remap from="collision_map" to="collision_map_occ" />
   <remap from="collision_map_update" to="collision_map_occ_update" />

   <param name="pointcloud_padd" type="double" value="0.00" />

   <param name="group" type="string" value="right_arm" />
   <param name="allow_valid_collisions" type="bool" value="true" />

   <param name="collision_map_safety_timeout" type="double" value="100000.0" />
   <param name="joint_states_safety_timeout" type="double" value="1.0" />
   <param name="tf_safety_timeout" type="double" value="1.0" />
   <param name="bounding_planes" type="string" value="0 0 1 -0.01" />

   <rosparam command="load" file="$(find pr2_arm_navigation_config)/config/robot_padding.yaml" />
  </node>
</launch>
